public/dorequest.php

```diff
@@ -162,25 +165,30 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
         }
         break;
 
+    // TODO require authentication?
     case "codenotes2":
         $response['CodeNotes'] = getCodeNotesData($gameID);
         $response['GameID'] = $gameID;
         break;
+
     case "gameid":
         $md5 = request()->input('m') ?? '';
         $response['GameID'] = getGameIDFromMD5($md5);
         break;
 
+    // TODO require authentication?
     case "gameslist":
         $consoleID = (int) request()->input('c', 0);
         $response['Response'] = getGamesListDataNamesOnly($consoleID);
         break;
 
+    // TODO require authentication?
```

Comment:
I'm not aware of any official uses for this. If it's being used by unofficial clients, we should provide a public API for this instead.

Reply:
This is indeed being used by third-party front-ends.